.gitignore

```diff
@@ -37,7 +37,15 @@ captures/
 
 # Intellij
 *.iml
-.idea/
```

Comment:
I think it is better to keep the whole .idea directory ignored, just in case a new file appears that we do not have control over. Since you have added the interesting files to the version control, they will be maintained, so there will be no problem.

What do you think?

Reply:
Yes :). I have not thought about that, but that should work. I'll check it out.